Chapters/01-zig-weird.qmd

```diff
@@ -159,13 +159,29 @@ So, everything you need to build a complex Zig project is the
 [^zig-build-system]: <https://ziglang.org/learn/overview/#zig-build-system>.
 
 
-The second generated file (`build.zig.zon`) is the Zig package manager configuration file,
-where you can list and manage the dependencies of your project. Yes, Zig has
-a package manager (like `pip` in Python, `cargo` in Rust, or `npm` in Javascript) called Zon,
-and this `build.zig.zon` file is similar to the `package.json` file
-in Javascript projects, or, the `Pipfile` file in Python projects,
-or the `Cargo.toml` file in Rust projects.
-
+The second generated file (`build.zig.zon`) is a JSON-like file, in which you can describe

```

Comment:
@pedropark99 you may still want to mention what the .zon extension stands for, it is indeed similar to JSON (JavaScript Object Notation) but instead of the JavaScript syntax it uses Zig syntax, it is "Zig Object Notation" (ZON)